pegasus/cache/i18n/en-US.yml

```diff
@@ -598,6 +598,7 @@
   volunteer_engineer_submission_distance_16: "10 miles"
   volunteer_engineer_submission_distance_24: "15 miles"
   volunteer_engineer_submission_distance_32: "20 miles"
+  volunteer_engineer_submission_distance_80: "50 miles"
```

Comment:
As @bethanyaconnor diagnosed (in Slack), the source of truth for these strings is the `I18n` gsheet, which explains why the string was subsequently [erased](https://github.com/code-dot-org/code-dot-org/commit/23a9c20325dea365cea69010d55795ce21af66db#diff-2810e2ff76b5aabd69e8a2fd9b527885a4f6218b0f83d2041fd2c1343d596102L601).  (It's obviously not so obvious that this will happen!)